### ISSUE TITLE
perceval.backends.core.git.EmptyRepositoryError is not handled

### ISSUE THREAD

**Issue by canasdiaz** [chaoss/grimoirelab-elk#553]
When the exception **perceval.backends.core.git.EmptyRepositoryError** is raised, the code in grimoirelab-elk crashes instead of handling the exception. The exception should be handled and an error message should be printed to error log.

Related issues: #550, #551, #552, #554, #555, #556

This is the traceback:
```
Error enriching ocean from git (https://github.com/symphonyoss/Apps-Integrations-GoogleCalendar.git): https://github.com/symphonyoss/Apps-Integrations-GoogleCalendar.git is empty
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 640, in enrich_backend
    enrich_count = enrich_items(ocean_backend, enrich_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 426, in enrich_items
    enrich_backend.update_items(ocean_backend, enrich_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/enriched/git.py", line 677, in update_items
    current_hashes = set([commit for commit in git_repo.rev_list()])
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/enriched/git.py", line 677, in <listcomp>
    current_hashes = set([commit for commit in git_repo.rev_list()])
  File "/usr/local/lib/python3.5/dist-packages/perceval/backends/core/git.py", line 964, in rev_list
    raise EmptyRepositoryError(repository=self.uri)
perceval.backends.core.git.EmptyRepositoryError: https://github.com/symphonyoss/Apps-Integrations-GoogleCalendar.git is empty
```

**Comment by valeriocos** [comment]
PR available https://github.com/chaoss/grimoirelab-elk/pull/549

**Comment by valeriocos** [comment]
Fixed with #549
